modules/services/playerctld.nix

```
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let cfg = config.services.playerctld;
+in {
+  meta.maintainers = maintainers.fendse;
+
+  options.services.playerctld = {
+    enable = mkEnableOption "playerctld daemon";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.playerctl;
+      defaultText = literalExample "pkgs.playerctl";
+      description = "The playerctl package to use";
```

```suggestion
      description = "The playerctl package to use.";
```

We use complete sentences in the description.

---

Missed that one, thank you